### PR TITLE
Add manager,HR : Routes, Controllers, Layouts

### PR DESCRIPTION
**Pull request by gggyyyamar**

### CODE REVIEW

**Comment by gggyyyamar** [comment]
Check on this

File: index.js
Comment (EvansMungai) [inline index.js:1]: This change is not necessary. I have edited it in the coming pull request. Only add the line that connect the manager routes to the server. The rest you can discard them

File: views/layouts/LeaveApplication.ejs
Comment (EvansMungai) [inline views/layouts/LeaveApplication.ejs:1]: Discard these changes, i have already worked on them.

File: server/routes/employeeDashboard.js
Comment (EvansMungai) [inline server/routes/employeeDashboard.js:1]: Good effort, however i have already worked on this entire file.

File: server/controllers/managerController.js
Comment (EvansMungai) [inline server/controllers/managerController.js:1]: Good work, will work on sessions and connecting it to the front end.

**Review by EvansMungai — APPROVED** [review]

**Pull request merged**